vsts/pipelines/validation.yml

```diff
@@ -17,7 +17,7 @@ jobs:
   displayName: Build and Test Build Images
   pool:
     name: OryxLinux
-  timeoutInMinutes: 300
+  timeoutInMinutes: 480
```

Comment:
should we leave it as `300`.

Reply:
I increased it because the building and testing of images is now taking a long time due to recent changes related to reducing layers of build images and also conda related tests. If we do not increase the timeout devops is killing off the job.